src/components/Makefile.comp

```diff
@@ -40,7 +40,7 @@ OPT= -g
 #OPT= -O3
 CFLAGS=-m32 -D__x86__ -Wall -Wextra -Wno-unused-parameter -Wno-unused-function -fno-stack-protector -fno-omit-frame-pointer -Wno-unused-variable -fvar-tracking $(INC_PATH) $(MUSLINC) $(LWIPINC) $(LUAINC) $(OPT) $(SHARED_FLAGS)
 CXXFLAGS=-fno-exceptions -fno-threadsafe-statics -Wno-write-strings $(CFLAGS)
-LDFLAGS=-m elf_i386
+LDFLAGS=-melf_i386
```

Comment:
Is this a useful/meaningful diff?

Reply:
I've tested on my systems and for some reason I've found `-m elf_i386` to not work

Reply:
Also, why not stay consistent with `-m32`? `-melf_i386` certainly works so I see no reason to introduce a space when we aren't using one anywhere else in the above code

Reply:
Linker, assember, and compiler might take different arguments.  Just make sure it works on all systems.